api/src/main/java/org/openmrs/module/pihcore/setup/InitializerSetup.java

```diff
@@ -42,7 +45,8 @@ public static void loadPreConceptDomains() {
             for (Loader loader : Context.getService(InitializerService.class).getLoaders()) {
                 if (!excludeList.contains(loader.getDomainName())) {
                     log.warn("Loading from Initializer: " + loader.getDomainName());
-                    loader.loadUnsafe(new ArrayList<>(), true);
+                    List<String> exclusionsForLoader = getExclusionsForLoader(loader, config);
+                    loader.loadUnsafe(exclusionsForLoader, true);
```

Comment:
Since you are re-opening this, we should really add this same logic here to the "loadPostConceptDomains" method, so that we have full coverage with consistent behavior.  Would you mind adding that in?

Reply:
Makes sense, will do.

Reply:
It doesn't look like you did this?  I don't see anything below in the PR @mogoodrich

Reply:
Right, I haven't done it yet... :)

Or, rather, I did something, but haven't pushed it up yet...